cli/src/main/java/com/devonfw/tools/ide/tool/vscode/Vscode.java

```diff
@@ -131,6 +131,7 @@ protected void configureToolArgs(ProcessContext pc, ProcessMode processMode, Pro
     pc.addArg("--user-data-dir=" + vsCodeConf);
     Path vsCodeExtensionFolder = this.context.getIdeHome().resolve("plugins/vscode");
     pc.addArg("--extensions-dir=" + vsCodeExtensionFolder);
+    pc.addArg(this.context.getWorkspacePath());
```

Comment:
Did you test this in combination with plugin installations?
This argument will be added to any invocation of `runTool` and that is not only used to start the IDE but also to install plugins. In the latter case this argument is IMHO undesired and might even break VScode.
In case I am right, you could make this conditional on `processMode` or you could move this extra arg to `run()` method (overriding it).

Reply:
Yes I just tested it and it worked out regardless. Anyways, I've found a bug not related to my change:
When vscode is already installed, plugins won't be reinstalled when a user manually deleted the plugins from the plugins/vscode- folder. Instead it just create an empty extensions.json- file.

Reply:
That is covered by #654.
Thanks for testing.